Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -125,7 +125,7 @@ namespace EMotionFX
                         ->DataElement(0, &EditorActorComponent::m_renderSkeleton,
                             "Draw skeleton", "Toggles rendering of skeleton.")
                         ->Attribute(AZ::Edit::Attributes::ChangeNotify, &EditorActorComponent::OnRenderFlagChanged)
-                        ->DataElement(0, &EditorActorComponent::m_renderBounds, "Draw bounds", "Toggles rendering of world space bounding boxes.")
+                        ->DataElement(0, &EditorActorComponent::m_renderBounds, "Draw bounds", "World Space AABBs. Teal: Static. Red: Bone Position. Blue: Mesh Vertices.")
```

Comment:
The problem here was the lack of space available in the tooltip. With a sentence it just ends up being cut off. So I was looking for a way to keep it as short as possible in order to fit all the colors into the tooltip. This way, when you see 3 aabbs being visualized, you can tell which one is which based on the color, and you don't have to leave the editor to find any documentation in order to know which one is which.

Reply:
Interesting, I thought the tooltip UI was resizing correctly to the length of the text. Tagging @cgalvan to raise awareness since this same problem would happen to other components.

Reply:
I would think there is a guideline on the amount of text a tool-tip should show. I would suggest asking for guidelines from UX.

Reply:
Not sure whether it was a bug that got fixed, or maybe I was mistaken in the first place. I've tested it locally and it's not getting cut off. I added a couple of line breaks, which also makes it easier to read.